site/src/components/roadmap/Roadmap.tsx

```diff
@@ -0,0 +1,246 @@
+  const filteredData = sortedRoadmapData.filter((item) => {
+    const matchesSearchQuery =
+      searchQuery === "" ||
+      item.text.toLowerCase().includes(searchQuery.toLowerCase());
+    return matchesSearchQuery;
+  });
+
+  return (
+    <Grid rowData={filteredData} style={{ height: 2000 }}>
+      <GridColumn
+        name="Name"
+        id="name"
+        defaultWidth={200}
+        getValue={(rowData) => rowData.text}
+      />
+      <GridColumn
+        name="Release"
+        id="location"
+        defaultWidth={150}
+        getValue={(rowData) => {
+          const formattedDate = formatDate(new Date(rowData.targetDate));
+          return formattedDate;
+        }}
+      />
+    </Grid>
+  );
+};
+
+
+
+export const CardView = ({ sortedRoadmapData, searchQuery }: CardViewProps) => {
+
+  return(
+    <GridLayout  style={{ maxWidth: "1000px", paddingBottom: "00px"}}  columns={3}  >
+
+    <div className={styles.column}>
+      <h2 className={styles.heading} style={{marginLeft: "22%"}}>Future <ProgressPendingIcon className={styles.icon} size={1.4}/></h2>
+        {  sortedRoadmapData.filter((item) => {
+        const startDate = item.startDate ? new Date(item.startDate) : null;
+        const today = new Date();
+        const isFutureItem = !startDate || startDate > today;
+        const matchesSearchQuery =
+          searchQuery === "" ||
+          item.text.toLowerCase().includes(searchQuery.toLowerCase());
+        return isFutureItem && matchesSearchQuery;
+      })
+          .map((item) => {
+            const formattedDate = formatDate(new Date(item.targetDate));
+
+            return (
+              <RoadmapCard className={styles.card} key={item.id}>
+                <H1 style={{ marginTop: "-10px" }} styleAs="h3">
+                  <a href={item.issueUrl}>
+                    <u>{item.text}</u>
+                  </a>
+                </H1>
+                <b>Due Date: </b>
+                <p className={styles.date}>{formattedDate}</p>
+              </RoadmapCard>
+            );
+          })}
+  </div>
+  <div className={styles.column}>
+    <h2 className={styles.heading} style={{marginLeft: "13%"}}>In Progress <ProgressInprogressIcon  className={styles.icon} size={1.4}/></h2>
+      {sortedRoadmapData.filter((item) => {
+  const startDate = new Date(item.startDate);
+  const targetDate = new Date(item.targetDate);
+  const today = new Date();
+  const isInRange = startDate <= today && today <= targetDate;
+  const matchesSearchQuery =
+  searchQuery === "" ||
+  item.text.toLowerCase().includes(searchQuery.toLowerCase());
+  return isInRange && matchesSearchQuery;
+        })
+        .map((item) => {
+          const formattedDate = formatDate(new Date(item.targetDate));
+
+    
+          return (
+            <RoadmapCard key={item.id}>
+              <H1 style={{ marginTop: "-10px" }} styleAs="h3">
+              <a href={item.issueUrl}>
+                    <u>{item.text} </u>
```

Comment:
why a `u`?

Reply:
just to give a link look, sine i think H1 doesn't include that and overrides the <a> underline

Reply:
Can you use a Link component instead of an anchor element and wrap the Link around the H1

Reply:
done